cmd/poller/collector/collector.go

```diff
@@ -540,6 +540,9 @@ func (me *AbstractCollector) LoadPlugins(params *node.Node, c Collector) error {
 			p = c.LoadPlugin(name, abc)
 			me.Logger.Debug().Msgf("loaded plugin [%s]", name)
 		}
+		if p == nil {
```

Comment:
good catch - wondering if a log message is warranted when p is nil. I haven't checked but it's likely the case that all callers of `c.LoadPlugin(name, abc)` log a msg before returning nil in which case also logging here may be redundant